### DISCUSSION TITLE
Ts function problem

### DISCUSSION THREAD

**Discussion by dimasrepo** [teamalgoritma/community#898] [Q&A] [answered]
Dear Team Algoritma,
Izin bertanya, saat ini saya sedang mencoba melakukan visualisasi pada model dimana dataset Scooty TS dari periode"2017-10-01 00:00:17 hingga "2017-12-02 23:59:20", mengapa data yang divisualisasikan hingga 2080 tidak sesuai dengan periode di data pada tanggal "2017-12-02 23:59:20" ? Bagaimana settingan agar sesuai kebutuhan berdasarkan hournya ? Berikut screenshotnya 
1. Frequency 12
![image](https://github.com/user-attachments/assets/df84a591-05a0-4a95-a442-d1bd50523857)
2. Frequency 24
![image](https://github.com/user-attachments/assets/22eb5d48-698e-48a5-b95e-f8d5f68fbd7a)
3. Autoplot Frequency 24
![image](https://github.com/user-attachments/assets/927571a7-97aa-4b62-9185-b0edc0f3e642)

**Comment by arkanayudhistira** [comment]
Dear Pak @dimasrepo,

Untuk menjawab pertanyaan dari Bapak, sebelumnya saya ingin memastikan, apakah Bapak sebelum melakukan pembuatan object time series sudah melakukan agregasi data terlebih dahulu? sehingga data memiliki interval yang tetap (dalam kasus ini data per-jam)

**Reply by dimasrepo** [reply]
![image](https://github.com/user-attachments/assets/1ecf4a5c-5aae-4afd-b186-7b719b9b62e2)

![image](https://github.com/user-attachments/assets/a9128010-c0df-4f56-b6e8-0baa36b5e083)

![image](https://github.com/user-attachments/assets/a3216f8e-2094-4cc5-8af8-2ea329d88642)

Sudah mas sudah di aggregasi , intervalnya juga sudah by hour sudah di input missing valuenya dengan padr  dengan na.locf juga.
Berikut saya coba rubah lagi syntaxnya tapi tetap yaa freqnya belum 24 jg hasil outputnya
![image](https://github.com/user-attachments/assets/31225492-9df4-4a37-ac4f-b36b8b55d460)

**Reply by arkanayudhistira** [reply, marked as answer]
Baik Pak @dimasrepo, terima kasih untuk jawabannya.

Untuk object time series tersebut tidak masalah pak untuk start dan end nya meskipun mungkin tidak sesuai dengan keinginan, karena object time series dari R hanya akan menampilkan pola frekuensi untuk data tertentu saja (seperti frekuensi 12 untuk data bulanan), sayangnya data per jam dengan frekuensi harian (24) tidak ditampilkan.

Meskipun begitu, kejadian tersebut tidak akan mempengaruhi proses kedepannya seperti modelling dan forecasting yang merupakan tujuan utama dari Capstone tersebut, jadi bisa langsung dilanjutkan saja pak untuk task nya.

Semoga membantu!

**Comment by dimasrepo** [comment]
Oh gitu ya mas saya kira eror soalnya, terimakasih untuk bantuannya

Dimas
